src/main/java/de/tu_darmstadt/cbs/emailsmpc/Study.java

```diff
@@ -1172,4 +1185,17 @@ private Message getShareMessage(int recipientId) throws IOException {
         Participant recipient = this.getParticipants()[recipientId];
         return new Message(getOwnId(), recipient, data.getMessage());
     }
+
+    /**
+     * Returns the exchange mode
+     * 
+     * @return
+     */
+    public ConnectionTypes getExchangeMode() {
```

Comment:
ConnectionTypes exchangeMode -> Connection type or exchange mode? Rename class or variable and method name.